### DISCUSSION TITLE
Only for Kubernetes?

### DISCUSSION THREAD

**Discussion by lonix1** [stern/stern#270] [Q&A] [answered]
Noob question. I was looking for a terminal log viewer for docker containers, and found this. Was quite excited.

But it looks like it's for Kubernetes only.

I'm not using Kubernetes, just plain docker. Is it useful for my use case as well?

**Comment by superbrothers** [comment, marked as answer]
stern is Kubernetes-focused; supporting the Docker Engine would require an entirely new set of features, which is not within the scope of this tool.

We apologize for not meeting your expectations.
